conf.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+const (
+	cliLogLevel = "log-level"
+	cliCertFile = "cert-file"
+	cliKeyFile  = "key-file"
+)
+
+type Conf struct {
+	CertFile string
+	KeyFile  string
+	LogLevel string
```

Comment:
I added it because it was part of the configuration file. I think it can re-write the LogLevel of rhc . But this change seemed out of the scope of this PR.

I'll remove and create a ticket with this as a feature.